flathub.json

```diff
@@ -0,0 +1,6 @@
+{
+  "automerge-flathubbot-prs": true,
+  "only-arches": ["x86_64"],
+  "publish-delay-hours": 0,
+  "skip-appstream-check": true
```

Comment:
```suggestion,
  "only-arches": ["x86_64"]
```

Reply:
I've removed all the options except only-arches. Was there a problem with that one too?